tests/integration_tests/alerts_tests.py

```diff
@@ -266,6 +266,9 @@ def test_operator_validator(setup_database):
     # Test passing with result that equals threshold
     assert operator_validator(alert2, '{"op": "==", "threshold": 55}') is True
 
+    # Test passing with result that equals decimal threshold
+    assert operator_validator(alert2, '{"op": ">", "threshold": 54.999}') is True
```

Comment:
This test is for the old alerts code base. This test can be added here: https://github.com/apache/superset/blob/master/tests/integration_tests/reports/commands_tests.py#L414

Optional (but would be great). Adding integration API tests to assert bad and good `validator_config_json`